NBXplorer.Client/Models/KeyPathInformation.cs

```diff
@@ -24,8 +24,18 @@ public KeyPathInformation(DerivationFeature feature, KeyPath keyPath, Derivation
 			KeyPath = keyPath;
 			Address = network.CreateAddress(derivationStrategy, keyPath, ScriptPubKey);
 		}
+		
+		public KeyPathInformation(Derivation derivation, DerivationSchemeTrackedSource derivationStrategy, DerivationFeature feature, KeyPath keyPath, NBXplorerNetwork network)
```

Comment:
can you remove this ctor? it is almost the same as above. (maybe the same with different order)